pyfakefs/fake_filesystem.py

```diff
@@ -1177,9 +1177,12 @@ def stat(self, entry_path, follow_symlinks=True):
             OSError: if the filesystem object doesn't exist.
         """
         # stat should return the tuple representing return value of os.stat
-        file_object = self.resolve(
-            entry_path, follow_symlinks,
-            allow_fd=True, check_read_perm=False)
+        try:
+            file_object = self.resolve(
+                entry_path, follow_symlinks,
+                allow_fd=True, check_read_perm=False)
+        except TypeError:
+            file_object = self.resolve(entry_path)
```

Comment:
this looks like another update that was made to pathlib (https://github.com/python/cpython/blob/main/Lib/pathlib.py#L1052)

not sure this is a particularly good way to handle it but it *does* seem to work

Reply:
Will leave it as is, I think it is good enough for now.